src/modules/dashboard/components/Extensions/ExtensionStatus.tsx

```diff
@@ -62,9 +63,19 @@ const ExtensionStatus = ({
   }
   return (
     <div className={styles.tagContainer}>
-      {!deprecatedOnly ? <Tag appearance={{ theme }} text={status} /> : null}
+      {!deprecatedOnly ? (
+        <Tag
+          appearance={{ theme }}
+          text={status}
+          data-test={`${kebabCase(status.defaultMessage)}-status-tag`}
+        />
+      ) : null}
       {installedExtension && installedExtension.details.deprecated ? (
-        <Tag appearance={{ theme: 'danger' }} text={MSG.deprecated} />
+        <Tag
+          appearance={{ theme: 'danger' }}
+          text={MSG.deprecated}
+          data-test="deprecated-status-tag"
```

Comment:
Any reason why you are going for the kebab case? I chose camel case and I also noticed that  both camel case and kebab case were used for the data-test in the dapp before (you can see leftover ones in some components). I don't mind either way, I think we should just agree on one and try to use it consistently across the dapp.
@arrenv @ArmandoGraterol  what do you think?

Reply:
@chinins I have so far only noticed CamelCase being used already in the Dapp, so, I have gone with that. I also prefer it, as with CamelCase you can double click to select the text vs. having to drag to select the text.